CMakeLists.txt

```diff
@@ -32,6 +32,8 @@ project(gnuradio CXX C)
 enable_testing()
 option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 
+OPTION(ENABLE_TESTING "Enable testing support" ON)
```

Comment:
This option is declared on line 381, with a dependency on Boost (all of it). I think if you move that code here & remove the dependency on Boost then this PR will work as desired. You'll have to `include(GrComponent)` before this, of course. Worth a try! Pretty sure CMake will complain if we declare 2 OPTION for the same variable ... might not error out, but it's not good coding style!